spec/factories/diffusion_histories.rb

```diff
@@ -10,6 +10,10 @@
       association :clinical_resource_hub
     end
 
+    # there are places in the application that expect a diffusion_history to have an associated
+    # diffusion_history_status, if you create a diffusion_history in rails console without one
+    # of the :with_status traits it will cause a breakage, ie /diffusion-map
```

Comment:
thanks for the helpful note here! We can add more error handling in diffusion map to our tech debt tickets